src/Symfony/Component/Notifier/Bridge/Telnyx/README.md

```diff
@@ -0,0 +1,24 @@
+Telnyx Notifier
+===============
+
+Provides [Telnyx](https://telnyx.com/) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+TELNYX_DSN=telnyx://API_KEY@default?from=FROM&profileId=PROFILE_ID
+```
+
+where:
+ - `API_KEY` is your telnyx API key.
+ - `FROM` is your sender.
+ - `PROFILE_ID` Unique identifier for a messaging profile. Required if sending via number pool or with an alphanumeric sender ID.
```

Comment:
```suggestion
 - `PROFILE_ID` identifier of your messaging profile at Telnyx. You need this in order to show a name to the recipient (e.g. "Symfony") instead of just the phone number.
```

Reply:
Done :)